mlflow/spark.py

```diff
@@ -618,14 +626,22 @@ def save_model(
                       The model signature can be :py:func:`inferred <mlflow.models.infer_signature>`
                       from datasets with valid model input (e.g. the training dataset with target
                       column omitted) and valid model output (e.g. model predictions generated on
-                      the training dataset), for example:
+                      the training dataset). If the signature describes model output then the
+                      corresponding columns will be included in the output schema from a
+                      `predict()` transformation. For example:
```

Comment:
```suggestion
                      the training dataset). The output schema defined in a signature will filter the model prediction 
                      return fields to return only those defined in the signature. For example:
```

Instead of doing the in-place examples of 'printing', can you mock up a .. code-block:: bash output that would actually show the schema to let users know that different outputs can be returned? It doesn't have to be more than a very simple example that shows "hey, you used to only be able to see the "prediction" column, but now you can see "x""

Reply:
i have added bash snippet of predicting from rest endpoint, is that what you meant? @BenWilson2